src/lib.rs

```diff
@@ -56,6 +56,7 @@ extern crate ring;
 extern crate rusoto_credential;
 extern crate rustc_serialize;
 extern crate serde;
+#[cfg(any(feature = "ecs", feature = "all", feature = "packagea", feature = "packageb"))]
```

Comment:
A more appropriate line that works:

`#[cfg(any(feature = "ecs", feature = "waf"))]`

Remaining work: add `feature = "serviceName"` for all services that use the macro.  This can be done by going through the Cargo file and running `cargo build --features serviceName` and seeing what breaks due to an error like this:

```bash
error: cannot find derive macro `Serialize` in this scope
    --> /Users/matthew/Documents/rusoto/target/debug/build/rusoto-762d371092257ef6/out/waf.rs:1217:30
     |
1217 | #[derive(Default,Debug,Clone,Serialize)]
```